developer-docs/bolt-api-servers.md

```diff
@@ -165,9 +166,18 @@ The metadata object is optional, and contains metadata about the task being run.
 - `parameters`: Object, *optional* - A JSON object whose keys are parameter names, and whose values are JSON objects with 2 keys:
     - `description`: String, *optional* - The parameter description.
     - `type`: String, *optional* - The type the parameter should accept.
+    - `sensitive`: Boolean, *optional* - Whether the task runner should treat the parameter value as sensitive
+    - `input_method`: String, *optional* - What input method should be used to pass params to task (stdin, environment, powershell)
 
 #### Files
-# TODO
+The files array contains details about the files the task needs as well as how to get them. Array items should be objects with the following keys:
+- `uri`: Object, *optional* - Information on how to request task files
```

Comment:
I think this is required, looking at the schema? https://github.com/puppetlabs/bolt/blob/master/lib/bolt_server/schemas/task.json#L69